### PR TITLE
add feature that allows to select a subset of samples used for sorting

### PR DESCRIPTION
**Pull request by steffenheyne**
within each group, add option --sortUsingSamples, provided samples are numbered from 1, numbers are checked if they are in range of existing samples. Nothing is changed yet in the matrix header line.

### CODE REVIEW

**Comment by steffenheyne** [comment]
maybe someone can review ist? thanks!

File: deeptools/computeMatrix.py [hunk 1]
```diff
@@ -370,7 +377,18 @@ def main(args=None):
     scores_file_list = args.scoreFileName
     hm.computeMatrix(scores_file_list, args.regionsFileName, parameters, blackListFileName=args.blackListFileName, verbose=args.verbose, allArgs=args)
     if args.sortRegions != 'no':
-        hm.matrix.sort_groups(sort_using=args.sortUsing, sort_method=args.sortRegions)
```
Comment (fidelram) [inline deeptools/computeMatrix.py]: I think here is args.sortUsingSamples instead of args.sortUsing

**Comment by dpryan79** [comment]
At some point it's easiest to do stuff like this after the fact. Have a look at the feature/riboseq_352 branch. There, I have a script (`scripts/subsetMatrix`) that does things like subset/merging matrices from computeMatrix. I foresee us making a new tool that only manipulates the output of computeMatrix.

**Comment by fidelram** [comment]
Seems to be OK. @steffenheyne I assume that you have run the code and that it gives you correct results.

**Comment by steffenheyne** [comment]
yes, output make sense to me.

**Pull request merged**